gfx/layers/wr/WebrenderPaintedLayer.cpp

```diff
@@ -65,7 +65,7 @@ WebRenderPaintedLayer::RenderLayer(wrstate* aWRState)
   }
   wr_dp_push_image(aWRState, toWrRect(rect), toWrRect(clip), NULL, key);
   Manager()->AddImageKeyForDiscard(key);
-  wr_pop_dl_builder(aWRState, bounds.x, bounds.y, bounds.width + bounds.x, bounds.height + bounds.y, &transform.components[0]);
+  wr_pop_dl_builder(aWRState, toWrRect(Rect()), toWrRect(bounds), &transform.components[0]);
```

Comment:
slightly confusing here - we pass ` toWrRect(bounds)` in place of an argument named `overflow`, and we pass `toWrRect(Rect())` for an argument named `bounds`